docs/release_notes/v1.7.0.md

```diff
@@ -0,0 +1,386 @@
+- **ADDED** Logging for sidecar pod name in Dapr Operator [3947](https://github.com/dapr/dapr/issues/3947)
+- **FIXED** Method invocation / Dapr automatically decodes %2f in URL [4008](https://github.com/dapr/dapr/issues/4008)
+- **FIXED** TLS connectivity to gRPC proxy functionality when --app-ssl is enabled [4048](https://github.com/dapr/dapr/issues/4048)
+- **ADDED** More metrics [4068](https://github.com/dapr/dapr/issues/4068)
+- **ADDED** logging for Dapr API calls over gRPC and HTTP [4119](https://github.com/dapr/dapr/issues/4119)
+- **ADDED** Nightly builds for Dapr [4124](https://github.com/dapr/dapr/issues/4124)
+- **FIXED** helm charts: increase failure threshold for liveness and readiness probes [4132](https://github.com/dapr/dapr/pull/4132)
+- **FIXED** dapr reconnect logic with rabbitmq results in "channel not initialized" [4143](https://github.com/dapr/dapr/issues/4143)
+- **ADDED** sidecar-injector health check port setting in helm chart [4174](https://github.com/dapr/dapr/issues/4174)
+- **FIXED** Configuration API, listen to all keys [4178](https://github.com/dapr/dapr/issues/4178)
+- **ADDED** HealthzPort option to injector helm chart - defaults to 8080 [4179](https://github.com/dapr/dapr/pull/4179)
+- **FIXED** Dapr retries forever when app is erroring on subscriptions [4185](https://github.com/dapr/dapr/issues/4185)
+- **FIXED** gRPC proxy connection closes after mTLS TTL passed [4197](https://github.com/dapr/dapr/issues/4197)
+- **ADDED** Dapr images on GitHub Registry in addition to Docker Hub [4199](https://github.com/dapr/dapr/issues/4199)
+- **FIXED** Multiple access control policies for a single app not working [4202](https://github.com/dapr/dapr/issues/4202)
+- **ADDED** New options available for the default secret-reader deployed by the dapr_rbac subchart. [4203](https://github.com/dapr/dapr/issues/4203)
+- **FIXED** Sidecar loads components instead of skipping if exists when operator stream reconnects [4212](https://github.com/dapr/dapr/issues/4212)
+- **FIXED** Update of components in Kubernetes mode was accidentally available and not working properly, now blocked. [4218](https://github.com/dapr/dapr/issues/4218)
+- **ADDED** Introduce state store component backed by Oracle Database or OCI Autonomous Database [4221](https://github.com/dapr/dapr/issues/4221
+- **FIXED** Condition in ReminderRename that would leave Reminder deleted. [4229](https://github.com/dapr/dapr/issues/4229)
+- **CHANGED** Automatic State Store Encryption to stable feature [4260](https://github.com/dapr/dapr/issues/4260)
+- **ADDED** `prometheus.io/probe` annotation [4272](https://github.com/dapr/dapr/issues/4272)
+- **ADDED** Jetstream state store [4276](https://github.com/dapr/dapr/pull/4276)
+- **FIXED** When the deployment selector is updated, dapr will get a "connection refused" error [4287](https://github.com/dapr/dapr/issues/4287)
+- **ADDED** Support for CockroachDB StateStore [4335](https://github.com/dapr/dapr/pull/4335)
```

Comment:
should be part of components contrib section ...

Reply:
It already is in contrib section -- this is just the PR for registering it in dapr/dapr. Maybe can be removed?

Reply:
removed that and jetsream.